src/StoreProvider.spec.lua

```diff
@@ -3,6 +3,7 @@ return function()
 
 	local Roact = require(script.Parent.Parent.Roact)
 	local Rodux = require(script.Parent.Parent.Rodux)
+	local folder = Instance.new("Folder")
```

Comment:
We should create this folder inside the test that uses it, since we don't want to re-use it across cases.